app/assets/javascripts/member-facing/backbone/fundraiser.js

```diff
@@ -1,12 +1,12 @@
-const ActionForm  = require('member-facing/backbone/action_form');
-const CurrencyMethods     = require('member-facing/backbone/currency_methods');
-const OverlayToggle       = require('member-facing/backbone/overlay_toggle')
-const BraintreeHostedFields = require('member-facing/backbone/braintree_hosted_fields');
-const GlobalEvents = require('shared/global_events');
+import mapValues from 'lodash/mapValues';
+import keyBy from 'lodash/keyBy';
+import Cookies from 'js-cookie';
```

Comment:
I believe JavaScript dependencies are best managed via npm in `package.json`. I'd prefer if we did the same with Backbone and other rails-assets gems since they're mostly just wrappers (for sprockets) around the actual npm packages.